api_tests/src/bitcoin_miner.ts

```diff
@@ -18,14 +18,15 @@ async function run(configFile: string) {
         port: config.rpcPort,
         username: config.username,
         password: config.password,
+        wallet: "miner",
     });
 
     // only coins after the first 101 are spendable
-    await client.generateToAddress(101, await client.getNewAddress());
+    const address = await client.getNewAddress();
```

Comment:
Who is using this address?

Reply:
I don't understand because you are generating a new address in this scope. You are then sending money to this address but the address is a local variable.

How do you then use the money sent to this address?

Or is it because this address is the one of a bitcoind wallet meaning you can then use the money by using `sendtoaddress` rpc call?

Reply:
That's the address of the miner. No need to know it and we can safely reuse. When minting Bitcoin the money will be sent from this address.

Reply:
Any danger in not reusing it?

To make it more descriptive, we could pass a "label" to this function call like "miner blockreward": https://bitcoin.org/en/developer-reference#getnewaddress

Reply:
I don't think this is necessary here. 
I opted for only 1 address because it's a speed improvement (1 rpc call vs 2 every second).